tests/acceptance/customCommands/findItemInFilesList.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Find an item in the files list. Scrolls down in case the item is not visible in viewport
+ */
+exports.command = async function (itemName) {
```

Comment:
feel like that should be part of the fileList Page object, that is an action that is done on/by the page

Reply:
Ok, will move